pkg/resources/manager/manager.go

```diff
@@ -472,6 +436,18 @@ func (r *resourceManager) getMachineClassAvailibilityAsString() string {
 	return removeSeparatorFromEnd(result)
 }
 
+func (r *resourceManager) getAvailableResources() core.ResourceList {
+	resourceList := make(core.ResourceList)
+
+	for _, s := range r.sources {
+		list := s.GetAvailableResource().DeepCopy()
```

Comment:
i recommend break reference directly in sources

Reply:
Done